libcxx/utils/libcxx/test/features.py

```diff
@@ -212,6 +212,35 @@ def _getAndroidDeviceApi(cfg):
           """,
         ),
     ),
+    # Some platform do not provide complete floating point environment.
+    Feature(
+        name="has-compolete-fenv",
+        when=lambda cfg: sourceBuilds(
+            cfg,
+            """
+            #include <fenv.h>
+
+            #if !( \
+                defined FE_DIVBYZERO \
```

Comment:
This feels a bit like a maintenance burden, if C adds new macros this need to be updated too.
Since we have not needed this before now I don't feel this solves a generic.
How about adding a comment and an `// UNSUPPORTED: pico-lib identifier` in these two tests instead?

Reply:
It used to make sense to me, as I am doing tests on more configurations of picolibc and some of them do provide all the necessary macros (e.g. armv7m with fpu). Now, I realize this is not that useful for libc++ itself, as only one configuration is tested in CI here.

Yet, still I will switch from XFAIL to UNSUPPORTED to cater the configurations, where this test would pass.